docs/install.md

```diff
@@ -1,14 +1,78 @@
 # SAT Installation
 
-## Install the SAT Product Stream
-
-Describes how to install the System Admin Toolkit (SAT) product stream.
-
-### Prerequisites
-
-- CSM is installed and verified.
-- There must be at least 2 gigabytes of free space on the manager NCN on which the
-  procedure is run.
+## Install and Upgrade Framework
+
+The Install and Upgrade Framework (IUF) provides commands which install,
+upgrade, and deploy products on systems managed by CSM. IUF capabilities are
+described in detail in the [IUF
+section](https://cray-hpe.github.io/docs-csm/en-14/operations/iuf/iuf/) of the
+[Cray System Management Documentation](https://cray-hpe.github.io/docs-csm/).
+The initial install and upgrade workflows described in the
+[HPE Cray EX System Software Stack Installation and Upgrade Guide for CSM
+(S-8052)](https://www.hpe.com/support/ex-S-8052) detail when and how to use
+IUF with a new release of SAT or any other HPE Cray EX product.
+
+This document **does not** replicate install, upgrade, or deployment procedures
+detailed in the [Cray System Management
+Documentation](https://cray-hpe.github.io/docs-csm/). This document provides
+details regarding software and configuration content specific to SAT which is
+needed when installing, upgrading, or deploying a SAT release. The [Cray
+System Management Documentation](https://cray-hpe.github.io/docs-csm/) will
+indicate when sections of this document should be referred to for detailed
+information.
+
+IUF will perform the following tasks for a release of SAT.
+
+- IUF `deliver-product` stage:
+  - Uploads SAT configuration content to VCS
+  - Uploads SAT information to the CSM product catalog
+  - Uploads SAT content to Nexus repositories
+- IUF `update-vcs-config` stage:
+  - Updates the VCS integration branch with new SAT configuration content if a
+    working branch is specified
+- IUF `update-cfs-config` stage:
+  - Creates new CFS management configurations with new SAT configuration content
+- IUF `prepare-images` stage:
+  - Creates updated NCN and compute node management images with new SAT content
+- IUF `management-nodes-rollout` stage:
+  - Boots NCNs with an image containing new SAT content
```

Comment:
It's important to say "management NCNs" because NCN just means non-compute nodes, which technically could include things like UANs, which is not what's being booted here. By saying "management NCNs", we're specifying only the nodes with the "management" role.

```suggestion
- Boots management NCNs with an image containing new SAT content
```

Reply:
Updating in next commit